pkg/archive/gzip_test.go

```diff
@@ -0,0 +1,88 @@
+package archive
+
+import (
+	"bytes"
+	"compress/gzip"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNewStabilizedGzipWriter(t *testing.T) {
+	tests := []struct {
+		name        string
+		input       []byte
+		header      gzip.Header
+		stabilizers []any
+		wantHeader  gzip.Header
+	}{
+		{
+			name:  "all stabilizers",
+			input: []byte("hello world"),
+			header: gzip.Header{
```

Comment:
Add a compression level to verify that it's set to gzip.NoCompression?

Reply:
Done.